src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,178 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static User validUser;
+    private User loginUserTest;
+    private User passwordUserTest;
+    private User ageUserTest;
+    private RegistrationService registrationService;
+
+    @BeforeAll
+    static void beforeAll() {
+        validUser = new User();
+        validUser.setLogin("login1");
+        validUser.setId(123456789L);
+        validUser.setAge(18);
+        validUser.setPassword("123456");
+    }
+
+    @BeforeEach
+    void setUp() {
+        registrationService = new RegistrationServiceImpl();
+        Storage.people.clear();
+
+        loginUserTest = new User();
+
+        passwordUserTest = new User();
+        passwordUserTest.setLogin("login1");
+
+        ageUserTest = new User();
+        ageUserTest.setLogin("login1");
+        ageUserTest.setPassword("123456");
+    }
+
+    @Test
+    void register_nullUser_notOK() {
+        assertThrows(InvalidUserDataException.class,
+                () -> registrationService.register(null));
+    }
+
+    @Test
+    void register_nullLogin_notOK() {
+        assertThrows(InvalidUserDataException.class,
+                () -> registrationService.register(loginUserTest));
+    }
+
+    @Test
+    void register_emptyLogin_notOK() {
+        loginUserTest.setLogin("");
+        assertThrows(InvalidUserDataException.class,
+                () -> registrationService.register(loginUserTest));
+    }
+
+    @Test
+    void register_3lengthLogin_notOk() {
+        loginUserTest.setLogin("abc");
+        assertThrows(InvalidUserDataException.class,
+                () -> registrationService.register(loginUserTest));
+    }
+
+    @Test
+    void register_5LengthLogin_notOk() {
+        loginUserTest.setLogin("abcde");
+        assertThrows(InvalidUserDataException.class,
+                () -> registrationService.register(loginUserTest));
+    }

```

Comment:
combine all checks you need to  `register_shortLogin_notOk`

Reply:
![image](https://github.com/mate-academy/jv-registration-validation/assets/77063748/e390703d-0293-4b0f-b98c-023dcac4b9c7)
The task clearly says to create multiple tests